components/region_cache_memory_engine/src/background.rs

```diff
@@ -243,6 +255,14 @@ impl BgWorkManager {
                             );
                         }
                     },
+                    recv(load_evict_ticker) -> _ => {
+                        if let Err(e) = scheduler.schedule(BackgroundTask::TopRegionsLoadEvict) {
```

Comment:
how to prevent TopRegionsLoadEvict running concurrently? That is the previous one has not finished, but the new task starts running again. Maybe need a flag indicating it's running or not.